src/main/java/laba1/Forecast.java

```diff
@@ -0,0 +1,41 @@
+package laba1;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Random;
+
+class Forecast {
```

Comment:
1. Create interfaces for each classes excepting  class which includs main() 
see https://www.jetbrains.com/help/idea/extract-interface.html
https://ru.stackoverflow.com/questions/136909/%D0%98%D0%BD%D1%82%D0%B5%D1%80%D1%84%D0%B5%D0%B9%D1%81%D1%8B-%D0%B2-%D0%9E%D0%9E%D0%9F-java-%D0%BF%D0%BE-%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%BE%D0%BC%D1%83